examples/widgets/popover_menu.nim

```diff
@@ -21,20 +21,32 @@
 # SOFTWARE.
 
 import owlkettle
+import owlkettle/[adw, playground]
 
 viewable App:
-  discard
+  sensitive: bool = true
+  sizeRequest: tuple[x, y: int] = (-1, -1) 
+  offset: tuple[x, y: int] = (0, 0)
+  position: PopoverPosition = PopoverBottom
 
 method view(app: AppState): Widget =
   result = gui:
     Window:
-      title = "Popover Menu Demo"
-      defaultSize = (300, 200)
-      
+      defaultSize = (800, 600)
```

Comment:
Revert title & default size

Reply:
Done, though I changed the width to 400. With the additional menubutton in the titlebar it otherwise lead to the title of the window not being fully shown.
![image](https://github.com/can-lehmann/owlkettle/assets/65814317/f21372a5-8476-4e33-9367-d7016ae10830)

For comparison 400:
![image](https://github.com/can-lehmann/owlkettle/assets/65814317/0ef31bbe-f5d2-478e-a246-c22f175c9515)